test/test_torch.py

```diff
@@ -16725,20 +16725,32 @@ def test_hardsigmoid(self, device, dtype):
 
     @dtypes(torch.float, torch.double)
     def test_silu(self, device, dtype):
-        inputValues = [-1000, -1, 0, 0.5, 1, 2, 1000]
-        expectedOutput = [0.0000, -0.2689, 0, 0.3112, 0.7312, 1.7616, 1000]
-        precision_4dps = 0.0002
+        input_np = np.random.randn(5, 8)
```

Comment:
I think it's better to just add silu to TestTorchMathOps (you can easily create np reference lambda for it) and remove this test altogether. If you don't want to do that, you can use `compare_with_numpy` helper function here to handle comparison.

Reply:
The issue is silu is a function in torch.nn.functional, not an attribute in tensor. It seems currently TestTorchMathMeta test the methods by getattr(x, opstr). Not sure if there is some proper way to do that.

Reply:
Yeah, ok, nevermind then.